src/assets/translations/sw-SW.json

```
@@ -2744,6 +2744,7 @@
       "All Recurring Deposits": "Amana Zote Zinazorudiwa",
       "All Savings": "Akiba Yote",
       "Savings Account Details": "Maelezo ya Akaunti ya Akiba",
+      "Create Savings Product": "Unda bidhaa ya akiba",
```

Biome (2.1.2)

[error] The key Create Savings Product was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)